src/test.js

```diff
@@ -719,15 +727,15 @@ function internalStart( test ) {
 
 function numberOfTests( module ) {
 	var count = module.tests.length;
-	while ( module = module.childModule ) {
+	while ( ( module = module.childModule ) ) {
```

Comment:
Thanks for the clarification. I'm fine with it as it might prevent unintended uses and the extra parenthesis flags it as intentional.